docker-build/README.md

```diff
@@ -6,6 +6,7 @@ This contains the builds of some images for KubeFATE to deploy FATE.
 - nginx
 - spark
 - python-spark
+- fate-test
```

Comment:
Why this 4 lines have different indents?

Reply:
I meam different with client on line 5

Reply:
This images list, I have modified.